content/actions/hosting-your-own-runners/configuring-the-self-hosted-runner-application-as-a-service.md

```diff
@@ -79,6 +79,8 @@ You can manage the runner service in the Windows **Services** application, or yo
    ```
 {% endmac %}
 
+The command takes an optional argument to install as a different user.
```

Comment:
I think it would be helpful to state the argument name/give an example. Is this accurate?:

````suggestion
The command takes an optional `user` argument to install the service as a different user.

```shell
./svc.sh install --user <em>USERNAME</em>
```
````